prog/download_boot_image.rb

```diff
@@ -26,9 +26,8 @@ def blob_storage_client
   end
 
   label def wait_draining
-    unless vm_host.vms.empty?
-      nap 15
-    end
+    nap 15 unless vm_host.vms_dataset.where(boot_image: image_name).all.empty?
```

Comment:
if you are working with datasets anyway, you can cut to the chase.

```suggestion
    nap 15 unless vm_host.vms_dataset.where(boot_image: image_name).empty?
```

Spares posgres forming a more advanced tuple:

```
4] clover-development(main)> DB.loggers << Logger.new($stdout)
=> [#<Logger:0x000075f602e14600
  @default_formatter=#<Logger::Formatter:0x000075f602e14470 @datetime_format=nil>,
  @formatter=nil,
  @level=0,
  @logdev=
   #<Logger::LogDevice:0x000075f602795ce8
    @binmode=false,
    @dev=#<IO:<STDOUT>>,
    @filename=nil,
    @mon_data=#<Monitor:0x000075f602e13f48>,
    @mon_data_owner_object_id=4240,
    @shift_age=nil,
    @shift_period_suffix=nil,
    @shift_size=nil>,
  @progname=nil>]
[5] clover-development(main)> Vm.dataset.empty?
I, [2024-04-04T13:19:28.655400 #49042]  INFO -- : (0.000566s) SELECT $1::int4 AS "one" FROM "vm" LIMIT 1; [1]
=> true
```

Reply:
interesting that sequel uses a parameter rather than a literal to simply render `1`, but :shrug:

Reply:
I also simplified tests too. Thanks